portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLAppServiceImpl.java

```diff
@@ -3454,7 +3458,7 @@ protected Folder moveFolder(
 	protected RepositoryProvider repositoryProvider;
 
 	private long[] _getCategoryIds(
-		String className, long classPK, List<Long> groupIds) {
+		String className, long classPK, long groupId, List<Long> groupIds) {
```

Comment:
Can you rename the `groupId` parameter to `repositoryId`? Its value is always a repository ID, and it would help to make the code a bit clearer.